pkg/trace/agent/agent.go

```diff
@@ -231,11 +232,13 @@ func (a *Agent) Process(p *api.Payload, sublayerCalculator *stats.SublayerCalcul
 				stats.SetSublayersOnSpan(subtrace.Root, subtraceSublayers)
 			}
 		}
-		sinputs = append(sinputs, &stats.Input{
-			Trace:     pt.WeightedTrace,
-			Sublayers: pt.Sublayers,
-			Env:       pt.Env,
-		})
+		if !p.ClientComputedStats {
```

Comment:
We'll still want to send sublayer stats to the concentrator

Reply:
Ok I think I got it now.